packages/apps/plugins/plugin-thread/src/translations.ts

```diff
@@ -16,6 +16,7 @@ export default [
         'message placeholder': 'Enter message...',
         'comment placeholder': 'Enter comment...',
         'settings standalone label': 'Enable standalone thread creation',
+        'anonymous label': 'Anonymous',
```

Comment:
Do we need this, vs generate name?

Reply:
We do need this as a UI-level detail — ThreadPlugin will substitute with a generated name so this is never encountered.